src/components/pages/donation_form/singlePageFromSections/PersonalDataSectionDonationReceipt.vue

```diff
@@ -200,16 +192,20 @@ interface Props {
 }
 
 const props = defineProps<Props>();
-const emit = defineEmits( [ 'previous-page' ] );
 const store = useStore();
 
 const { addressType, addressTypeName } = useAddressType( store );
 const { addressSummary, inlineSummaryLanguageItem } = useAddressSummary( store );
 const mailingList = useMailingListModel( store );
 const { receiptNeeded, showReceiptOptionError } = useReceiptModel( store );
 const countryWasRestored = ref<boolean>( false );
-const pageRef = ref<HTMLElement>( null );
 defineExpose( { focus: (): void => pageRef.value.focus() } );
+const scrollToPaymentSection = () => {
+	const scrollIntoViewElement = document.getElementById( 'donation-page-form-section-payment' );
```

Comment:
true, but I think I would leave it like that because it is clearer in the domain and the code might be easier to read ("scroll back to payment section")